daemon/cmd/ipam.go

```diff
@@ -208,7 +209,7 @@ func (d *Daemon) allocateDatapathIPs(family types.NodeAddressingFamily, fromK8s,
 		option.Config.IPAM == ipamOption.IPAMAlibabaCloud ||
 		option.Config.IPAM == ipamOption.IPAMAzure) && result != nil {
 		var routingInfo *linuxrouting.RoutingInfo
-		routingInfo, err = linuxrouting.NewRoutingInfo(result.GatewayIP, result.CIDRs,
+		routingInfo, err = linuxrouting.NewRoutingInfo(logging.DefaultSlogLogger, result.GatewayIP, result.CIDRs,
```

Comment:
Would it be worth adding `log *slog.Logger` to `Daemon` and pass that around in these places?

Reply:
Yes but not now. I have that locally already. I'm just kind of pushing PRs on a bottoms up approach and them I'll replace all of these default loggers with daemon's in the final PR.